pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -182,92 +197,27 @@ def __init__(self,
         self.verbose     = verbose
         self.mod_wake    = mod_wake
         self.attempt     = 1
+        self.floating    = floating
                                         
                                         
-        if self.verbose>0: print(f'Checking inputs...', end='\r')
+        if self.verbose>0: print('Checking inputs...', end='\r')
```

Comment:
All formatting changes should be submitted as separated commits in separate PRs.